lib/content/routes.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * @fileoverview Content system routing functions
+ *
+ */
+const config = require('config');
+const types = require('punchcard-content-types');
+const _ = require('lodash');
+
+const database = require('../database');
+const utils = require('../utils');
+
+/**
+ * All Content-Types Landing page
+ * @param {object} res - HTTP Response
+ * @param {object} allTypes all content types available to the CMS
+ */
+const all = (res, allTypes) => {
+  res.render('content/home', {
```

Comment:
I don't think we should continue that practice here. Those init scripts all build on each other in a row. These routes are stand-alone in respect to each other.

res.render is not typically returned (https://expressjs.com/en/guide/routing.html). I think it's appropriate in promises, but not if there is not promise.